src/screens/SSICredentialsRequiredScreen/index.tsx

```diff
@@ -151,24 +161,30 @@ const SSICredentialsRequiredScreen: FC<Props> = (props: Props): JSX.Element => {
         ? undefined
         : () => onItemPress(itemInfo.item.id, availableCredentials.get(itemInfo.item.id)!, itemInfo.item.purpose);
 
+    const checkIsMatching = (itemInfo: any, selectedCredentials: any, pex: any, getOriginalVerifiableCredential: any) => {
```

Comment:
Don't introduce any. We have types for these

Reply:
Uhm in our code you really should not use chatgpt for these types of fixes, tickets